python/src/opendp/_lib.py

```diff
@@ -287,16 +287,16 @@ def proven(function): # pragma: no cover
         source_dir = os.path.dirname(inspect.getfile(function))
         absolute_proof_path = os.path.abspath(os.path.join(source_dir, matched_path))
 
-
-        # split the path at the extrinsics directory
-        extrinsics_path = os.path.join(os.path.dirname(__file__), "_extrinsics")
-        relative_proof_path = os.path.relpath(absolute_proof_path, extrinsics_path)
+        # TODO: Is special handling for extras still needed?
```

Comment:
Yes, I believe this is still necessary- it has to do with links being relative from the repository root.